setup.py

```diff
@@ -20,7 +20,10 @@
 
 import os.path
 import subprocess
-import urllib
+try:
+    from urllib.request import urlretrieve
+except ImportError:
+    from urllib import urlretrieve
```

Comment:
Add a comment indicating this is for version compatibility so we can strip it out later.

Reply:
Done